lib/rubocop/cop/style/super_with_args_parentheses.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Style
+      # Enforces the presence of parentheses in `super` containing arguments.
```

Comment:
Might be good to extend the description here - e.g. mentioning that `super` is a keyword and that's why it needs a separate cop from the existing one for methods. There was also a bit of discussion in the past here https://github.com/rubocop/ruby-style-guide/issues/703